### DISCUSSION TITLE
reset then insert w/ field array bug?

### DISCUSSION THREAD

**Discussion by fadi-george** [react-hook-form/react-hook-form#3210] [Q&A] [answered]
Steps:
- wait 1 sec for reset to kick in
- click append
Clicking append should duplicate the item but on newer versions it doesnt.

v6.8.6
https://codesandbox.io/s/react-hook-form-686-reset-insert-8r1fm

v6.9.4
https://codesandbox.io/s/react-hook-form-694-reset-insert-forked-bpyun?file=/src/index.js

**Comment by bluebill1049** [comment, marked as answer]
thanks @fadi-george raise an issue as well, please 🙏

**Reply by fadi-george** [reply]
Done as well, in the meantime I had to use a hidden input type w/ unique id.
